README.md

```diff
@@ -6,12 +6,45 @@ for the API endpoints can be found in their
 [Confluence](https://alamos-support.atlassian.net/wiki/spaces/documentation/pages/1683226637/Monitoring-Schnittstelle).
 
 # Usage
+
+## Docker
+
+```sh 
+docker run -d \
+    -p 9865:9865 \ 
+    -e FE2_EXPORTER_HOST=alamos-fe2-server \
+    -e FE2_EXPORTER_PORT=83 \
+    -e FE2_EXPORTER_SSL=false \
+    -e FE2_EXPORTER_HOST=Gz8NcB6aKTp2uQ+Y1oVhRA== \
+    bouni/prometheus-fe2-exporter:latest 
```

Comment:
Probably not a good idea to link your personal image here

Reply:
I'll change that once codemonauts publish an official image on Dockerhub